course04/homework00/shankarvasudevan/sortedarray/src/sarray.h

```diff
@@ -0,0 +1,45 @@
+#include <array>
+#include <iostream>
```

Comment:
This should never be included as part of a container header. It's not needed and should not be here. iostream is quite heavy.